tool/vfx/vfxSection.h

```diff
@@ -1171,13 +1200,14 @@ class SectionVertexInputBinding : public Section {
 
   static void initialAddrTable() {
     StrToMemberAddr *tableItem = m_addrTable;
-    INIT_STATE_MEMBER_NAME_TO_ADDR(SectionVertexInputBinding, binding, MemberTypeInt, false);
-    INIT_STATE_MEMBER_NAME_TO_ADDR(SectionVertexInputBinding, stride, MemberTypeInt, false);
-    INIT_STATE_MEMBER_NAME_TO_ADDR(SectionVertexInputBinding, inputRate, MemberTypeEnum, false);
+    INIT_STATE_MEMBER_NAME_TO_ADDR(SectionVertexInputBinding, binding, uint32_t, MemberTypeInt, false);
```

Comment:
Please stick to using unsigned (not uint32_t) in this change. For uint8_t, consider if we could replace them with unsigned as well. I don't think we need explicit bit-width types.